src/sentry/static/sentry/app/views/performance/compare/utils.tsx

```diff
@@ -0,0 +1,771 @@
+import {SentryTransactionEvent} from 'app/types';
+import {RawSpanType, SpanType} from 'app/components/events/interfaces/spans/types';
+import {
+  parseTrace,
+  generateRootSpan,
+  isOrphanSpan,
+  toPercent,
+} from 'app/components/events/interfaces/spans/utils';
+
+export function isTransactionEvent(event: any): event is SentryTransactionEvent {
+  if (!event) {
+    return false;
+  }
+
+  return event?.type === 'transaction';
+}
+
+export type DiffSpanType =
+  | {
+      comparisonResult: 'matched';
+      span_id: SpanId; // baselineSpan.span_id + regressionSpan.span_id
```

Comment:
Style thing but why is snake case mixed in here in a few places?

Reply:
The snake case `span_id` and `parent_span_id` comes from the span data payload received.